README.md

```diff
@@ -7,3 +7,49 @@ Zip files uploaded to said bucket must be prefixed with "mi_" for Management Inf
 `mi_opn2001a_01012020_1200.zip`
 
 Appropriate zip file metadata messages are published to a Pub/Sub topic for MiNiFi to consume and transfer the zip files on-premises via NiFi.
+
+<b>Please post refactored and new Survey messages as examples in Confluence: https://collaborate2.ons.gov.uk/confluence/display/QSS/Blaise+5+Publish+PubSub+Topic+for+NiFi<b>
+
+##Local Setup
+
+Clone the project locally:
+
+```
+git clone https://github.com/ONSdigital/blaise-publish-bucket-metadata.git
+```
+
+Create a virtual environment:
```

Comment:
Very minor, but poetry will auto-generate you a virtual env when you run poetry install if one doesn't exist

Reply:
safe 👊